node/config/def.go

```diff
@@ -364,9 +364,11 @@ func DefaultCurioConfig() *CurioConfig {
 			SingleCheckTimeout:    Duration(10 * time.Minute),
 		},
 		Ingest: CurioIngestConfig{
-			MaxQueueSDR:   8, // default to 8 sectors before sdr
-			MaxQueueTrees: 0, // default don't use this limit
-			MaxQueuePoRep: 0, // default don't use this limit
+			MaxQueueDealSector: 8, // default to 8 sectors open(or in process of opening) for deals
+			MaxQueueSDR:        0, // default don't use this limit
```

Comment:
We probably should do some simulations around which buffers should be the main limiters; IIUC the theory says that we proobably want a small buffer before our main bottleneck/cost, which is usually SDR, and that buffer should be the main source of backpressure

Reply:
You are right here. I have added back the default of 8 and update the back pressure to account for SDR. Even if we have 0 open deal sectors, we should produce back pressure if SDR queue is full. This should account for CC sectors being sealed with deal sectors.